recipe/meta.yaml

```diff
@@ -32,11 +32,11 @@ requirements:
     - numpy >=1.13.3
     - six >=1.10.0
     - protobuf >=3.4.0
-    - tensorboard >=1.8.0, <1.9.0
+    - tensorboard 1.9
```

Comment:
Perhaps that should that be `1.9*`? i.e. any patch release in the `1.9` version.

Reply:
`1.9` also matches `1.9.*`. It's better than `>=1.9.0,<1.10` though in that the latter would also match `1.10.0rc1 ` or similar.